### DISCUSSION TITLE
TFT 35 e3 3.0 and SKR e3 turbo

### DISCUSSION THREAD

**Discussion by samia3d** [teeminus/NoTouchScreenFirmware#46] [Ideas]
I did not see any discussion on this combo on the installation page. with a bit of tinkering, I got the No touch screen firmware for the tft35 e3 to work with the SKR e3 turbo. let me know if I missed this info already listed but figured I would share in case not. 
the following worked when placed in my printer.cfg and having the EXP1 on the e3 turbo connected to EXP3 on the screen 
```
[board_pins]
aliases:
    # EXP1 header
    EXP1_1=P2.8,  EXP1_3=P0.19, EXP1_5=P0.20, EXP1_7=P0.17, EXP1_9=<GND>,
    EXP1_2=P0.16, EXP1_4=<RST>, EXP1_6=P0.15, EXP1_8=P0.18, EXP1_10=<5V>

[display]
lcd_type: st7920
cs_pin: EXP1_7
sclk_pin: EXP1_6
sid_pin: EXP1_8
encoder_pins: ^EXP1_5, ^EXP1_3
click_pin: ^!EXP1_2
```

**Comment by teeminus** [comment]
I added your config to the wiki. Thank you very much :)
